test/Elastic.Apm.AspNetCore.Tests/AspNetCoreMiddlewareTests.cs

```diff
@@ -84,6 +91,14 @@ public async Task HomeSimplePageTransactionTest()
 			transaction.Context.Request.Url.HostName.Should().Be("localhost");
 			transaction.Context.Request.Url.Protocol.Should().Be("HTTP");
 
+			if (_agent.ConfigurationReader.CaptureHeaders)

```

Comment:
Same as with the other else block: do we have scenarios where it is actually tested?